command-expansion-server/package.json

```diff
@@ -14,16 +14,18 @@
   "license": "ISC",
   "dependencies": {
     "@microsoft/tsdoc": "^0.13.2",
+    "@nasa-jpl/aerie-ampcs": "^1.0.0",
     "express": "^4.17.2",
-    "multer": "^1.4.4",
+    "graphql": "^16.3.0",
```

Comment:
This is unused, so lets remove it

Reply:
Once I rebased with `develop` it was removed